conan/tools/apple/apple.py

```diff
@@ -18,15 +18,15 @@ def to_apple_arch(arch, default=None):
             'armv7k': 'armv7k'}.get(arch, default)
 
 
-def get_apple_sdk_name(conanfile):
+def get_apple_sdk_name(settings):
```

Comment:
If this is a tool that could be called from the outside, it better receive ``conanfile`` as argument, in case we need to inject some ``[conf]`` at some point. If it is private at the moment, good, it can use ``settings``, but lets recall not to make this public or document it.